README.md

```diff
@@ -1,2 +1,61 @@
-# html-css-capstone
-A project for completion of HTML and CSS block at Microverse
+<h2>Psych Conference</h2>
+
+![Screenshot (45)](https://user-images.githubusercontent.com/69966775/124003051-63747900-d9ce-11eb-830e-f6d0c6c5144c.png)
+
+<h6>First Capstone Project</h6>
+
+Additional description about the project and its features.
+
+<h1>Built With</h1>
+
+- HTML
+- CSS
+- JS
+
+## Live demo
+
+This is the [live demo](https://sly-cloud.github.io/html-css-capstone/) of my capstone project.
+
+## Presentation Video
+
+This [video](https://www.loom.com/share/7117d7bbad67485dbe1d6c4bf97434a1) explains the aspects of the website and the idea behind it
+
```

Comment:
Thank you for making such a nice video presentation :clap: 

- [ ] However, you're expected to highlight a piece of code that you find interesting or proud of making. This is not present in your presentation, and it will be nice if you can do this. You may decide to either make another presentation that covers all you've talked about in this plus the missing feature, or you can record another video of about a minute describing the missing feature :blush: